R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -1042,6 +1046,19 @@ test_that("mutate(), rename() and names()", {
   names(newDF2) <- c("newerName", "evenNewerAge")
   expect_equal(length(names(newDF2)), 2)
   expect_equal(names(newDF2)[1], "newerName")
+
+  transformedDF <- transform(df, newAge = -df$age, newAge2 = df$age / 2)
+  expect_equal(length(columns(transformedDF)), 4)
+  expect_equal(columns(transformedDF)[3], "newAge")
+  expect_equal(columns(transformedDF)[4], "newAge2")
+  expect_equal(first(filter(transformedDF, transformedDF$name == "Andy"))$newAge, -30)
+
+  # ensure the proper signature is used - otherwise this will fail to run
+  attach(airquality)
```

Comment:
Hmm so this is testing whether `transform` on local data frames continue to work ? its an interesting idea -- could you update the comment on top to say `test if transform on local data frames works. ensures that proper signature ...`.